static/app/components/searchSyntax/parser.tsx

```diff
@@ -283,6 +283,10 @@ type InvalidFilter = {
    * This may be multiple filter types.
    */
   expectedType?: FilterType[];
+  /**
+   * The invalid reason type
+   */
+  type?: InvalidReason;
```

Comment:
If we're going to do this for one let's do it for all of them and not make this optional.

Reply:
@evanpurkhiser, I have successfully added types for all other invalid reasons. However, implementing a dropdown menu with an invalid message for each of them is not a straightforward task and would require careful planning and additional time. While I experimented with the file https://github.com/getsentry/sentry/blob/master/static/app/components/smartSearchBar/index.tsx, several updates were made to cover all the invalid reasons, resulting in the breaking of many tests. Initially, the aim was to simply inform users that wildcard usage is not supported. If we really need to add support for the reasons for this change, I believe we might have to postpone the addition of this extra information for now.